ister_gui.py

```diff
@@ -644,6 +672,247 @@ def validate_condition(self):
         return True
 
 
+class ChooseAction(ProcessStep):
+    """UI to select installation path (Install, Shell, or Repair)"""
+    def __init__(self):
+        super(ChooseAction, self).__init__()
+        self.choices = {
+            'Install': 'Install          - Install Clear Linux OS',
+            'Repair': 'Repair           - Attempt to repair host OS',
+            'Shell': 'Shell            - Open shell in host OS',
+            'Update Installer': 'Update Installer - Update installer to '
```

Comment:
All this does is run `swupd update` in the installer image. Don't we want our users to always have the most up-to-date installer? If this fails the user can do as you suggest and grab a new installer image. I don't see how updating the image they are on would cause any bugs, unless swupd fails in a strange way.